src/sirhelpers.c

```diff
@@ -314,25 +314,24 @@ int _sir_fopen(FILE* restrict* restrict streamptr, const char* restrict filename
 
 struct tm* _sir_localtime(const time_t* restrict timer, struct tm* restrict buf) {
     if (_sir_validptr(timer) && _sir_validptr(buf)) {
-#if defined(__HAVE_STDC_SECURE_OR_EXT1__)
+#if defined(__HAVE_STDC_SECURE_OR_EXT1___)
```

Comment:
2 underscores at the end, not 3, unless it is defined that way elsewhere in which case it's wrong there too

Reply:
Fixed